src/wizard/javascript/sveltekit/with-error-monitoring-and-performance.md

```diff
@@ -0,0 +1,71 @@
+---
+name: SvelteKit
+doc_link: https://docs.sentry.io/platforms/javascript/guides/sveltekit/
+support_level: production
+type: framework
+---
+
+## Install
+
+Configure your app automatically with the [Sentry wizard](https://docs.sentry.io/platforms/javascript/guides/sveltekit/#install).
+
+```bash
+npx @sentry/wizard -i sveltekit
+```
+
+## Configure
+
+Sentry wizard will automatically patch your application to configure the Sentry SDK:
+
+- Create or update `src/hooks.client.js` and `src/hooks.server.js` with the default `Sentry.init` call and SvelteKit hooks handlers.
+- Update `vite.config.js` to add source maps upload and auto-instrumentation via Vite plugins.
+- create `.sentryclirc` and `sentry.properties` files with configuration for sentry-cli (which is used when automatically uploading source maps).
+
+Alternatively, you can also [set up the SDK manually](https://docs.sentry.io/platforms/javascript/guides/sveltekit/manual-setup/).
+
+**Configure the Sentry SDK**:
+
+To configure the Sentry SDK on the client-side, edit the `Sentry.init` options in `hooks.client.(js|ts)`:
+
+```javascript
+import * as Sentry from '@sentry/sveltekit';
+
+Sentry.init({
+  dsn: '___PUBLIC_DSN___',
+
+  // Performance Monitoring integration
+  integrations: [new Sentry.BrowserTracing()],
+  // Capture 100% of the transactions. Adjust this value in production as necessary:
+  tracesSampleRate: 1.0,
```

Comment:
```suggestion
  tracesSampleRate: 1.0, // Capture 100% of the transactions. Adjust this value in production as necessary:
```

Reply:
Hmm no strong opinion but I personally don't like super long comments after code and would rather have it above the concerned statement. But I realize that this is inconsistent with other guides (nextjs at least). So happy to change it if you think this needs to be changed.

Reply:
yes please just for the sake of consistency

Reply:
Done [2dbfc1c](https://github.com/getsentry/sentry-docs/pull/6867/commits/2dbfc1cf24562ec0c7a47fdd5c4ea213b753155f)